236: Theme community pages

bronzehedwick opened the pull request:
This PR addresses all the required pieces of #236, except the membership list block in the sidebar. I'm still working on that, so set this to Draft, but wanted to put up everything else so folks could take a peek and provide any feedback.

bronzehedwick committed:
236: Theme community nodes

quicksketch committed:
236: Fixing rendering of community and protocol entities to use templates and view modes.

bronzehedwick committed:
236: Rename communities item class

bronzehedwick committed:
236: Fix community entity class path

bronzehedwick committed:
236: Group community full page fields

bronzehedwick committed:
236: Change community base selector

bronzehedwick committed:
236: Layout community pages

bronzehedwick committed:
236: Style banner image

bronzehedwick committed:
236: Use horizontal cards for featured items

bronzehedwick committed:
236: Fix sidebar display

bronzehedwick committed:
236: Style sidebar items

bronzehedwick committed:
236: Remove Protocol from sidebar

bronzehedwick committed:
236: Put protocols back

bronzehedwick committed:
236: Remove unused variables

bronzehedwick committed:
236: Add browse by community

bronzehedwick committed:
236: Give content columns a max width

bronzehedwick committed:
236: Style featured items like community callouts

bronzehedwick committed:
236: Use teaser display for featured content

michael-wynne-wsu commented:
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr787-kkxcusirygon9q9wtmd732nrvzzdn22d.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/67eedbe7e026d48ed34d06dd

michael-wynne-wsu commented:
@bronzehedwick this looks REALLY good. A few requests/notes though (anything not noted below is good as is!)

1) Unless a long enough description is added, that main section doesn't seem to pad out and the right sidebar shifts inward.
![Screenshot 2025-04-03 at 12 18 00 PM](https://github.com/user-attachments/assets/b1ed3fc0-f8f3-43ee-9b41-59e492ca3bd5)

2) Group role block. As presented it won't mean anything to users. I think ideally the wording would be like...
- Community role
- "You are a [community manger/community member/community affiliate]."

3) The featured content previews don't feel quite right. 
- I think we'd want a 3 or 4 column grid to fit the page, not 2.
- Something is off with the `>` placement (we could get rid of that).
- Content with non-image assets may not be using the desired preview/player. I have NOT tested all content types or all media.
- Thumbnails aren't showing when logged out.

![Screenshot 2025-04-03 at 12 20 27 PM](https://github.com/user-attachments/assets/e639f795-8691-407b-84e7-d8a683d21dcb)

![Screenshot 2025-04-03 at 12 23 25 PM](https://github.com/user-attachments/assets/79b42c4b-4534-4e43-92d8-3c06fb3ae10c)

bronzehedwick committed:
236: Content well has min width

bronzehedwick committed:
236: Remove community button arrow

bronzehedwick committed:
236: Use 3 up grid

bronzehedwick committed:
236: Fix rule order SASS deprecation error

bronzehedwick committed:
236: Improve iframe embed styles

michael-wynne-wsu commented:
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr787-kkxcusirygon9q9wtmd732nrvzzdn22d.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/67eedbe7e026d48ed34d06dd

bronzehedwick committed:
236: Rename OG Group block

michael-wynne-wsu commented:
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr787-kkxcusirygon9q9wtmd732nrvzzdn22d.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/67eedbe7e026d48ed34d06dd

bronzehedwick committed:
236: Re-write OG Role block description

Use desired text. Use a preprocess function instead of a template
override for both title and description.

bronzehedwick committed:
236: Allow all featured content aspect ratios

Do not crop or cover, let the asset go through.

bronzehedwick committed:
236: Theme word list teasers

michael-wynne-wsu commented:
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr787-kkxcusirygon9q9wtmd732nrvzzdn22d.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/67eedbe7e026d48ed34d06dd

michael-wynne-wsu commented:
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr787-kkxcusirygon9q9wtmd732nrvzzdn22d.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/67eedbe7e026d48ed34d06dd

michael-wynne-wsu commented:
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr787-kkxcusirygon9q9wtmd732nrvzzdn22d.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/67eedbe7e026d48ed34d06dd

michael-wynne-wsu commented:
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr787-kkxcusirygon9q9wtmd732nrvzzdn22d.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/67eedbe7e026d48ed34d06dd

michael-wynne-wsu commented:
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr787-kkxcusirygon9q9wtmd732nrvzzdn22d.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/67eedbe7e026d48ed34d06dd

michael-wynne-wsu commented:
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr787-kkxcusirygon9q9wtmd732nrvzzdn22d.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/67eedbe7e026d48ed34d06dd

bronzehedwick committed:
236: Document community items don't show extra document link

michael-wynne-wsu commented:
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr787-kkxcusirygon9q9wtmd732nrvzzdn22d.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/67eedbe7e026d48ed34d06dd

bronzehedwick committed:
236: Fix featured item dictionary word styles

bronzehedwick committed:
236: Print audio files in dictionary words teaser

michael-wynne-wsu commented:
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr787-kkxcusirygon9q9wtmd732nrvzzdn22d.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/67eedbe7e026d48ed34d06dd

michael-wynne-wsu commented:
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr787-kkxcusirygon9q9wtmd732nrvzzdn22d.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/67eedbe7e026d48ed34d06dd

bronzehedwick commented:
I should have addressed all your feedback @michael-wynne-wsu. Do you want to take another look? I also set this PR as ready for review, to hopefully get this chunk of work in separate from the membership list, which I can do in a follow up PR.

michael-wynne-wsu commented:
@bronzehedwick I'll spin it up right now

michael-wynne-wsu reviewed (approved):
I'm extremely happy with this, I don't see any changes needed!

bronzehedwick commented:
Great! I'll merge this and keep working on the membership list in a follow up.
